mbtiles/src/mbtiles.rs

```diff
@@ -39,6 +40,82 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct LevelDetail {
```

Comment:
I keep stumbling on the word "level" here.  Lets rename it to `ZoomInfo` or `PerZoomInfo` or something like that?  The `Statistics` could also have a `zoom_info_list: Vec<ZoomInfo>` ?  Naming is hard, no good ideas here.

Reply:
I was struggling here either.  Is `ZoomStats`  and `zoom_stats_list` Ok?  If not, let's pick `ZoomInfo` firstly.

Reply:
sure, zoom stats works